falcosidekick/CHANGELOG.md

```diff
@@ -5,6 +5,28 @@ numbering uses [semantic versioning](http://semver.org).
 
 Before release 0.1.20, the helm chart can be found in `falcosidekick` [repository](https://github.com/falcosecurity/falcosidekick/tree/master/deploy/helm/falcosidekick).
 
+## 0.3.0
+
+### Major Changes
+
+* Add `AWS S3` output  
+* Add `GCP Storage` output
+* Add `RabbitMQ` output
+* Add `OpenFaas` output
+
+### Minor Changes
+
+* Use higher level Writer api for Kafka
+* Reorder imports to follow good practices
+* Prevent misleading error message when CUSTOMFIELDS env var is set
+* Use Events v2 API for PagerDUty output
+
+### Fixes
+
+* Fix outputformat when using fields or text in Slack output
+* Fix HTML template for SMTP output
+
```

Comment:
the changes are related to `falcosidekick` itself, not the chart, you can remove them

Reply:
Okay